bitnami/haproxy/Chart.yaml

```
@@ -25,4 +25,4 @@ maintainers:
 name: haproxy
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/haproxy
-version: 0.8.7
+version: 0.9.0
```

According to [semver](https://semver.org) this new version should be a patch version because there is no new features.

```suggestion
version: 0.8.8
```